src/components/calcite-shell-panel/calcite-shell-panel.tsx

```diff
@@ -36,7 +36,12 @@ export class CalciteShellPanel {
   /**
    * This sets limits the height of the content area. It only applies when detached is true.
    */
-  @Prop({ reflect: false }) detachedScale: CalciteScale = "m";
+  @Prop({ reflect: true }) detachedScale: CalciteScale = "m";
```

Comment:
Maybe we should rename this to `heightScale`?

Reply:
Hee hee...yeah. That's what I was thinking.
Although, it only applies when `detached` is true.
So maybe it's `detachedHeightScale`?

Reply:
Let me know what you think makes sense.

Reply:
What if we just document that it will only affect a detached shell panel and call it `heightScale`?

Reply:
I'm cool with that.

Reply:
I'll update this property and the doc.